.rubocop.yml

```diff
@@ -12,8 +12,10 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
+AllCops:
+  TargetRubyVersion: 2.1
 Metrics/LineLength:
-  Max: 100
+  Max: 130
```

Comment:
is this only for not breaking the one liner?

Reply:
100 is too small.

```
Offenses:

Demo/EarlGreyExample/Podfile:23:8: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
target "EarlGreyExampleTests" do
       ^^^^^^^^^^^^^^^^^^^^^^
Demo/EarlGreyExample/Podfile:30:8: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
target "EarlGreyExampleSwiftTests" do
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
Demo/EarlGreyExample/Podfile:36:1: C: Layout/TrailingBlankLines: 1 trailing blank lines detected.
gem/spec/earlgrey_yaml_spec.rb:6:101: C: Metrics/LineLength: Line is too long. [105/100]
    allow(podfile).to receive(:defined_in_file) { File.join(__dir__, '/fixtures/earlgrey_yaml/Podfile') }
                                                                                                    ^^^^^
gem/lib/earlgrey/cli.rb:24:101: C: Metrics/LineLength: Line is too long. [110/100]
    method_option :scheme,  aliases: '-s', type: :string, required: false, desc: 'EarlGreyTestTarget.xcscheme'
                                                                                                    ^^^^^^^^^^
gem/lib/earlgrey/extensions/analyzer_extensions.rb:35:101: C: Metrics/LineLength: Line is too long. [119/100]
          EarlGrey.add_earlgrey_copy_files_script native_target, framework_ref if config[EarlGreyYaml::ADD_BUILD_PHASE]
                                                                                                    ^^^^^^^^^^^^^^^^^^^
gem/lib/earlgrey/extensions/analyzer_extensions.rb:36:101: C: Metrics/LineLength: Line is too long. [109/100]
          EarlGrey.add_earlgrey_framework native_target, framework_ref if config[EarlGreyYaml::ADD_FRAMEWORK]
                                                                                                    ^^^^^^^^^
gem/lib/earlgrey/extensions/earlgrey_yaml.rb:19:101: C: Metrics/LineLength: Line is too long. [116/100]
    abort "ERROR: earlgrey_gem.yml references missing targets: #{missing_targets}".red unless missing_targets.empty?
                                                                                                    ^^^^^^^^^^^^^^^^
gem/lib/earlgrey/extensions/earlgrey_yaml.rb:30:101: C: Metrics/LineLength: Line is too long. [105/100]
    abort "ERROR: earlgrey_gem.yml contains unknown keys: #{unknown_keys}".red unless unknown_keys.empty?
                                                                                                    ^^^^^
gem/lib/earlgrey/configure_earlgrey.rb:319:101: C: Metrics/LineLength: Line is too long. [129/100]
        settings[HEADER_SEARCH_PATHS] << CARTHAGE_HEADERS_IOS unless settings[HEADER_SEARCH_PATHS].include?(CARTHAGE_HEADERS_IOS)
                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gem/lib/earlgrey/configure_earlgrey.rb:345:101: C: Metrics/LineLength: Line is too long. [127/100]
      target.frameworks_build_phase.add_file_reference framework_ref, true unless linked_frameworks.include? EARLGREY_FRAMEWORK
                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
gem/lib/earlgrey/configure_earlgrey.rb:384:101: C: Metrics/LineLength: Line is too long. [107/100]
        raise "Unable to locate #{swift_fallback} file at path #{src_swift}." unless File.exist?(src_swift)
                                                                                                    ^^^^^^^
```